packages/create-vite/template-solid/jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
```

Comment:
I'm not sure about this file, we don't use it for other templates expect svelte.
What are the benefits?

Reply:
I'm not sure but it was in the original template, I'm not very sure what is it for but my guess it that it makes it possible to use the syntax that solid uses for the state (signal) like `count()`. I'll check it out and tell you what I find tho.

Reply:
I guess we can just remove it. nothing happened when I did.